ts/webui/src/components/trial-detail/search/SearchParameterConditions.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Stack, PrimaryButton, Dropdown, IDropdownOption } from '@fluentui/react';
+import { trialJobStatus } from '../../../static/const';
+import { getFiterConditionString } from '../../../static/function';
+import { searchConditonsGap } from '../../modals/ChildrenGap';
+
+// This file is for filtering trial parameters and trial status
+
+function SearchParameterConditions(props): any {
+    const { parameter, searchFilter, dismiss, changeSearchFilterList, updatePage, setSearchInputVal } = props;
+    const isStatus = parameter === 'StatusNNI';
+    const [trialParameterOperatorVal, setTrialParameterOperatorVal] = useState(getInputsVal()[0]);
+    const [trialStatusOperator, setTrialStatusOperator] = useState(getInputsVal()[1]);
+    const [firstInputVal, setFirstInputVal] = useState(getInputsVal()[1]);
+    const [secondInputVal, setSecondInputVal] = useState(getInputsVal()[2]);
+    const operatorList = isStatus ? ['=', '≠'] : ['between', '>', '<', '=', '≠'];
```

Comment:
I think this check should be by type.

* In case of string: equal, not equal.
* In case of number: between, less, greater, equal, ...

Reply:
think about user could input values to search trials, so only parse data type on searching trial function.
search filter list only store origin user input

Reply:
So what's the current support for string?

Reply:
Please refer to `learning_rate`:(demo link: http://13.77.78.63:8000/detail)
![image](https://user-images.githubusercontent.com/61399850/119095288-adb82100-ba44-11eb-8c2f-56862512adad.png)
![image](https://user-images.githubusercontent.com/61399850/119095331-bad51000-ba44-11eb-8d9a-8963e59cfe58.png)

Reply:
Oh. This looks awful. Can we make this unsupported if it's too hard to implement.

(I mean directly removing `learning_rate` from the dropdown menu)

Reply:
delete the support of choice parameter is easy to do...
I'll demo the latest progress in the meeting to confirm the final version!